Add leaflet

Hiroshi3080 opened the pull request:
<!---

何を変更したのか、以下について記載してください。
Please describe what has been changed and describe the following.

-->

# Changes/変更内容

- リーフレットがあることの紹介と、リーフレットタイトルの2行を追加

# Reason or background for the change/変更した理由や背景

- これまでなかったIAESTE] Japan の紹介リーフレットを制作した。プリント配布の他、ｐｄｆファイルをホームページに掲載して閲覧・ダウンロードができるようにする。

# Desired release date and time/リリース希望日時

- リリース日時の指定はしない

Hiroshi3080 committed:
Update about.md

新規制作した広報リーフレットを掲載

Hiroshi3080 commented:
次にリンク先のファイル　leaflet-IAESTE-Japan-v2025.pdf　をGithubに入れようとしました。

https://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static/files　の下に　
about  のホルダーがあればそこに入れるのですがabout  のホルダーはありません。　
これは、現行の　about.md に紐づくファイルがなかったためと思います。
この場合、どうやってファイルを渡せばいいですか？

dymmt commented:
> 次にリンク先のファイル　leaflet-IAESTE-Japan-v2025.pdf　をGithubに入れようとしました。
> 
> 
> 
> https://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static/files　の下に　
> 
> about  のホルダーがあればそこに入れるのですがabout  のホルダーはありません。　
> 
> これは、現行の　about.md に紐づくファイルがなかったためと思います。
> 
> この場合、どうやってファイルを渡せばいいですか？

## ファイル名について
大文字と小文字が入り混じっているので
'leaflet-ja-v20250327.pdf'にしませんか？

iaeste.or.jp 内では可能な限りファイル名はケバブケースに従って整理しています。

参考: https://developer.mozilla.org/ja/docs/Glossary/Kebab_case

## about フォルダの作成方法

1. まず最初に hugo/static/files/ 配下にこれからアップロードしたいファイルを1つアップロードしてください。

2. 以下の説明通りにファイル名を変更する状態にします
https://docs.github.com/ja/repositories/working-with-files/managing-files/renaming-a-file

3. ファイル名を変更する際 about/leaflet-ja-v20250327.pdf と入力すれば about ファイルの中に pdf が移動します

Hiroshi3080 committed:
Update about.md

リーフレットのファイルの名称変更

Hiroshi3080 committed:
Add files via upload

Hiroshi3080 committed:
Update about.md

Hiroshi3080 committed:
Update about.md

ファイル名をabout/leaflet-ja-v20250327.pdf に変更

dymmt commented on hugo/content/ja/about.md:
```diff
@@ -26,6 +26,8 @@ IAESTE は、 The International Association for the Exchange of Students for Tec
 日本では、1964年 (昭和39年) に任意団体として発足した、国際学生技術研修協会が活動を開始し、1982年 (昭和57年) からは社団法人日本国際学生技術研修協会、2012年 (平成24年) 以降は一般社団法人日本国際学生技術研修協会として、各国との間で累計 7,000 人以上の学生を交換してきました。
 
 海外へ研修に行く研修生の募集、日本へ研修に来る研修生のための歓迎プログラム実施など、国内における IAESTE 活動は、学生ボランティアと会員大学が中心となって実施されています。
+当協会（IAESTE Japan）の最近の活動については以下のリーフレットをご覧ください.
```
行末尾がピリオド "." になっています。
ページ全体の文調に合わせて読点 "。" に変更をお願いします。

Hiroshi3080 committed:
Update about.md

文末をピリオドに

Hiroshi3080 commented:
リーフレットのファイル名を'leaflet-ja-v20250327.pdf'　に変更し、文末を「。」にしました。

「about フォルダの作成方法」を見て行いました。
１．まず最初に hugo/static/files/ 配下にこれからアップロードしたいファイルを1つアップロードしてください。
　　　⇒アップロードしhttps://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static　に
　　　　'leaflet-ja-v20250327.pdf'　があることを確認しました。

２．以下の説明通りにファイル名を変更する状態にします
　　https://docs.github.com/ja/repositories/working-with-files/managing-files/renaming-a-file
　　　⇒読んでこの通りにやろうとしました。
　　　　が、「③ファイル名フィールドで、ファイルの名前を新しいファイル名に変更します。 」のところを見ると
　　　　ここのファイル名フィールドで変更するのは　'about.md' を他の名前にすることと考えられます。
　　　　about.md　の名前を変えていいのか？　と思って止まってしまいました。

「ファイル名を変更する際 about/leaflet-ja-v20250327.pdf と入力すれば about ファイルの中に pdf が移動します」
と山元さんが書いています。この意味は
https://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static/files　
の下に　about　のホルダーができる ということだと思いますが、いろいろ試してもabout　のホルダーは出来ませんでした。また、Firebase　でも　Not Found　になりました。

済みません、私がどこか勘違いしていますか？

dymmt commented:
> リーフレットのファイル名を'leaflet-ja-v20250327.pdf'　に変更し、文末を「。」にしました。
> 
> 「about フォルダの作成方法」を見て行いました。 １．まず最初に hugo/static/files/ 配下にこれからアップロードしたいファイルを1つアップロードしてください。 　　　⇒アップロードし[https://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static　に](https://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static%E3%80%80%E3%81%AB) 　　　　'leaflet-ja-v20250327.pdf'　があることを確認しました。
> 
> ２．以下の説明通りにファイル名を変更する状態にします 　　https://docs.github.com/ja/repositories/working-with-files/managing-files/renaming-a-file 　　　⇒読んでこの通りにやろうとしました。 　　　　が、「③ファイル名フィールドで、ファイルの名前を新しいファイル名に変更します。 」のところを見ると 　　　　ここのファイル名フィールドで変更するのは　'about.md' を他の名前にすることと考えられます。 　　　　about.md　の名前を変えていいのか？　と思って止まってしまいました。
> 
> 「ファイル名を変更する際 about/leaflet-ja-v20250327.pdf と入力すれば about ファイルの中に pdf が移動します」 と山元さんが書いています。この意味は [https://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static/files　](https://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static/files%E3%80%80) の下に　about　のホルダーができる ということだと思いますが、いろいろ試してもabout　のホルダーは出来ませんでした。また、Firebase　でも　Not Found　になりました。
> 
> 済みません、私がどこか勘違いしていますか？

https://github.com/iaeste-japan/www/tree/Hiroshi3080-patch-6/hugo/static/files を見ると

files/leaflet-ja-v20250327.pdf になっています。
leaflet-ja-v20250327.pdf をリネームして `about/leaflet-ja-v20250327.pdf` にしてみてください。
スペース無し、全て半角でお願いします。

このコメントでも全角スペースや全角英数が混じっているので、名前変更でも全角混じりが問題の原因になっていることを疑っています。

Hiroshi3080 commented:
改めて全角交じりのご指摘ありがとうございます。もっと気を付けます。

「leaflet-ja-v20250327.pdf をリネームして about/leaflet-ja-v20250327.pdf にしてみてください。」というのは、
①about.md のテキストを変更するという事でしたら、既に行いました。
②元のleaflet-ja-v20250327.pdfをGithubに入れる前にabout/leaflet-ja-v20250327.pdf　にする方法は、ファイル名に / スラッシュ）が使えないのでダメでした。

どうやってリネームすればいいのでしょうか？

教えていただいたhttps://docs.github.com/ja/repositories/working-with-files/managing-files/renaming-a-file 
は、hugo/static/filesにあるような pdfファイルの名前を変更するのでなく、about.md のようなcontentファイルの変更に使うように思われました。

dymmt commented:
> ①about.md のテキストを変更するという事でしたら、既に行いました。

それは about.md が参照する先を示すもの (ハイパーリンク) です。ファイルの置き場所には影響しません

> ②元のleaflet-ja-v20250327.pdfをGithubに入れる前にabout/leaflet-ja-v20250327.pdf　にする方法は、ファイル名に / スラッシュ）が使えないのでダメでした。

ファイル名を変更する対象は以下です
https://github.com/iaeste-japan/www/blob/Hiroshi3080-patch-6/hugo/static/files/leaflet-ja-v20250327.pdf


[ファイル名を変更する - GitHub Docs](https://docs.github.com/ja/repositories/working-with-files/managing-files/renaming-a-file) を参考にして `leaflet-ja-v20250327.pdf` を `about/leaflet-ja-v20250327.pdf` に変更してみてください。

以前記載した通りに素直にやってみてください。書いてある通りです

> 1. まず最初に hugo/static/files/ 配下にこれからアップロードしたいファイルを1つアップロードしてください。
> 
> 2. 以下の説明通りにファイル名を変更する状態にします
> https://docs.github.com/ja/repositories/working-with-files/managing-files/renaming-a-file
> 
> 3. ファイル名を変更する際 about/leaflet-ja-v20250327.pdf と入力すれば about ファイルの中に pdf が移動します

Hiroshi3080 committed:
Rename hugo/static/files/leaflet-ja-v20250327.pdf to hugo/static/files/about/leaflet-ja-v20250327.pdf

Hiroshi3080 committed:
Update about.md

pdf ファイルの名前の前についた about/ を削除

Hiroshi3080 committed:
Update about.md

Hiroshi3080 committed:
Add files via upload

WEBサイトからリーフレットのｐｄｆに飛ぶところまで行きましたが、エラー表示が出でpdっファイルが開きません。
現在の3月27日付けのファイルは元の17MBを圧縮したものです。
代わりに、圧縮前のファイルを3月28日の日付で使うことにします。

Hiroshi3080 committed:
Update about.md

Hiroshi3080 committed:
Update about.md

Hiroshi3080 commented:
ようやく分かりました。ありがとうございます。いろいろ試行錯誤しましたが、
何とかデモ画面でリンク先ファイルを含めて意図通りであることが確認できました。
本番への反映をよろしくお願いします。

dymmt commented on hugo/static/files/about/leaflet-ja-v20250327.pdf:
このファイルは2KBしかデータ量がないので圧縮時点で破損しているようです。

本番リリースにこのファイルは公開する必要がないため、削除をお願いします。

Hiroshi3080 commented:
leaflet-ja-v20250327.pdfのファイルは壊れてはいないと思います。
https://github.com/iaeste-japan/www/commit/388548231ee12f2553bc1046af4d49450e83dca2の画面で　
leaflet-ja-v20250327.pdfの容量が約3メガと表示され、また、ファイルを開いて表示させることが出来ました。
しかし、デモ画面上で開こうとするとｐｄｆファイルに関するエラーが出たのは確かです。　PC上では開くのですが。

私も不要となった leaflet-ja-v20250327.pdfを削除しようといろいろな画面を開いてみました。
が、ファイルを「削除」する機能が見つかりませんでした。どのようにして削除したらいいでしょうか？

dymmt commented:
> leaflet-ja-v20250327.pdfのファイルは壊れてはいないと思います。 [388548231ee12f2553bc1046af4d49450e83dca2の画面で　](https://github.com/iaeste-japan/www/commit/388548231ee12f2553bc1046af4d49450e83dca2の画面で　) leaflet-ja-v20250327.pdfの容量が約3メガと表示され、また、ファイルを開いて表示させることが出来ました。 しかし、デモ画面上で開こうとするとｐｄｆファイルに関するエラーが出たのは確かです。　PC上では開くのですが。

GitHubにアップロードされているファイルの詳細を見るとファイルが破損しています。2 Bytesしかデータがありません。

https://github.com/iaeste-japan/www/blob/Hiroshi3080-patch-6/hugo/static/files/about/leaflet-ja-v20250327.pdf

ファイル破損の根拠は以下の表示です。
> 0 lines (0 loc) · 2 Bytes

> leaflet-ja-v20250327.pdfの容量が約3メガと表示され、また、ファイルを開いて表示させることが出来ました。

ご自身のPC上では壊れていないようですが、アップロードされているファイルは壊れているのが事実です。
アップロードが中途半端に失敗したのかもしれません。圧縮の有無は問題ではないように思われます。

圧縮した状態のファイルを `leaflet-ja-v20250328.pdf` にして

https://github.com/iaeste-japan/www/blob/Hiroshi3080-patch-6/hugo/static/files/about/leaflet-ja-v20250328.pdf 

にアップロードすれば上書きできます。

> 私も不要となった leaflet-ja-v20250327.pdfを削除しようといろいろな画面を開いてみました。 が、ファイルを「削除」する機能が見つかりませんでした。どのようにして削除したらいいでしょうか？

以下を参照ください。
- https://docs.github.com/ja/repositories/working-with-files/managing-files/deleting-files-in-a-repository#deleting-a-file

些細な質問はAIに聞いてみると自己解決できると思います。
- [GitHub で GitHub Copilot に質問をする
](https://docs.github.com/ja/enterprise-cloud@latest/copilot/using-github-copilot/copilot-chat/asking-github-copilot-questions-in-github)

Hiroshi3080 committed:
Delete hugo/static/files/about/leaflet-ja-v20250327.pdf

ご教示ありがとうございました。
ようやく分かりました。

Hiroshi3080 committed:
Add files via upload

dymmt committed:
Rename a file
